_monitoring-your-cluster/pa/rca/shard-hotspot.md

```diff
@@ -0,0 +1,102 @@
+---
+layout: default
+title: Hot shard identification
+parent: Root Cause Analysis
+grand_parent: Performance Analyzer
+nav_order: 30
+---
+
+## Shard hotspot identification
+
+Hot shard identification Root Cause Analysis (RCA) lets you can identify a hot shard within an index. A hot shard is an outlier that consumes more resources than other shards and may lead to poor indexing and search performance. The hot shard identification RCA monitors the following metrics:
+
+- CPU utilization
+- Heap allocation rate
+
+Although the metric that you should monitor depends on the cluster configuration and the workload, refer to the following list for common operations which may lead to increased resource consumption:
+
+- Bulk requests: High heap allocation rate.
+- Search requests: High CPU utilization
+- Complex queries: High CPU utilization and high heap allocation rate.
+- Document updates: High CPU utilization and high heap allocation rate.
```

Comment:
This is generic and not related to Shard HotSpot.  Instead, we can add about the cluster workload using `_routing` parameter and/or custom document ID, mapping to a segment on disk. Repeated access to this segment(in turn OpenSeach shard) leading to more resource consumption and thus becoming a Hot Shard. 

We can discuss more on the phrasing of this.